packages/lib/src/prod/shared-production.ts

```diff
@@ -59,9 +59,7 @@ export function prodSharedPlugin(
       // Cannot emit chunks after module loading has finished, so emitFile first.
       if (parsedOptions.prodShared.length && isRemote) {
         this.emitFile({
-          fileName: `${
-            builderInfo.assetsDir ? builderInfo.assetsDir + '/' : ''
```

Comment:
Will the `__federation_fn_import` file be generated in different locations after the folder(builderInfo.assetsDir) is deleted?

Reply:
I've double-checked: it will be placed next to the other `__federation_shared_***.js` files.